psi4/src/psi4/libmints/eri.h

```diff
@@ -67,6 +67,41 @@ typedef struct ShellPair_typ {
     double** overlap;
 } ShellPair;
 
+/**
+ * \ingroup MINTS
+ * Structure to hold precomputed gaussian product information
+ */
+typedef struct PrimPairScreen_typ {
+    //!x, y, z coordinate of gaussian
+    double P[3];
+    //! Distance between P and shell i center
+    double PA[3];
+    //! Distance between P and shell j center
+    double PB[3];
+    //! alphas for both centers
+    double ai, aj;
+    //! gamma (ai + aj)
+    double gamma;
+    //! Contraction coefficients
+    double ci, cj;
+    //! Overlap between primitives
+    double overlap;
+} PrimPairScreen;
```

Comment:
There's nothing wrong with the C style struct here, but I think writing it as 
```
struct PrimPairScreen {
....
};
```
is a little more canonical for our C++ style.

Reply:
Got it, thanks!